### DISCUSSION TITLE
Units in Force RMSD lcurve.out?

### DISCUSSION THREAD

**Discussion by TinacciL** [deepmodeling/deepmd-kit#1264] [Q&A] [answered]
I do not find the information about the unit for the RMSD for Forces in the ```lcurve.out``` file.

Is it the RMSD for the whole system [eV/Å] or is normalised for the number of atoms [eV/Å/N]? 

Thanks

**Comment by wanghan-iapcm** [comment, marked as answer]
Please refer to eq 1 in PRL 120, 143001 (2018)
